tools/gn

```diff
@@ -226,6 +226,10 @@ def setup_rbe(args):
   # care of starting and stopping the compiler proxy.
   running_on_luci = os.environ.get('LUCI_CONTEXT') is not None
 
+  # Don't use racing on CI.
```

Comment:
Are there docs that explain what racing means in the context of RBE? Would be handy to link them here maybe.

Reply:
Or add a See: RbeConfig enum maybe?

Reply:
Done.